tests/phpunit/tests/functions/wpFuzzyNumberMatch.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Test wp_fuzzy_number_match().
+ *
+ * @group functions.php
+ * @covers ::wp_fuzzy_number_match
+ */
+class Tests_wpFuzzyNumberMatch extends WP_UnitTestCase {
+
+	/**
+	 * @dataProvider data_wp_fuzzy_number_match
+	 *
+	 * @param $expected
+	 * @param $actual
+	 * @param $precision
+	 * @param $result
+	 *
+	 * @return void
+	 */
+	public function test_wp_fuzzy_number_match( $expected, $actual, $precision, $result ) {
+		$this->assertEquals( $result, wp_fuzzy_number_match( $expected, $actual, $precision ) );
+	}
+
+	/**
+	 * @return array[]
+	 */
+	public function data_wp_fuzzy_number_match() {
+
+		return array(
+			1            => array(
```

Comment:
The first three keys should be enclosed within single quotes and would benefit from the type being included for consistency with the other datasets:

```
1 should be '1 int'
2 should be '2 int'
3 should be '3 int'
```